packages/app/src/components/Settings/TableRow.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import moment from 'moment';
+
+
+const TableRow = (props) => {
+    const { grantCycle, onEdit, show, onResultsClick } = props;
+
+    const formatDateString = date => {
+        const year = date.slice(0, 4);
```

Comment:
This function is a duplicate of another one above. This should be refactored into a helper method.